File: front/src/services/rest-services/kitchenuser.service.js
import Vue from 'vue'

const PATH_GET = 'getAllKitchenUsers'
const PATH_CREATE = 'createKitchenUser'
const PATH_UPDATE = 'updateKitchenUser'
const PATH_DELETE = 'deleteKitchenUserByID?id='

const KitchenUserService = {
// es6 sintaksa
  getAll: () => Vue.prototype.$axios.get(`${PATH_GET}`),
  createUser: (user) => Vue.prototype.$axios.post(`${PATH_CREATE}`, user),
  updateUser: (user) => Vue.prototype.$axios.put(`${PATH_UPDATE}`, user),
  deleteUser: (id) => Vue.prototype.$axios.delete(`${PATH_DELETE}${id}`)
}

export default KitchenUserService

File: back/src/main/java/me/fit/smartkitchen/service/api/KitchenUserService.java
package me.fit.smartkitchen.service.api;

import java.util.List;

import me.fit.smartkitchen.model.KitchenUser;

public interface KitchenUserService {

	public void createKitchenUser(KitchenUser user);

	public void updateKitchenUser(KitchenUser user);
	
	public void deleteKitchenUserByID(Long id);

	public List<KitchenUser> getAllKitchenUsers();

	public List<KitchenUser> getKitchenUsersByID(Long id);

}
